mmocr/core/evaluation/utils.py

```diff
@@ -136,23 +136,34 @@ def points2polygon(points):
     return plg(point_mat)
 
 
-def poly_intersection(poly_det, poly_gt, invalid_ret=0, return_poly=False):
+def poly_intersection(poly_det, poly_gt, invalid_ret=None, return_poly=False):
```

Comment:
I don't think so . Your suggestion may be a good practice if we have more than 3 functions implementing similar geometry computations. However, we only implement `union` and `intersection` for now so it is just an over-design.

Reply:
You are right and it is may over-design. My suggestion bases on two reason:it can avoid the two places change and it can expend other feature like different backend for intersection and union

Reply:
We should keep the implementation simple and make the function name as straightforward as possible. Focusing too much on the extensibilities of two basic utils will just increase the code complexity as of now, given that their implementations are already short, very unlikely to change, and hard to generalize. Also, changing the interface would be a BC-breaking change that should be avoided.